### ISSUE TITLE
now user day wise report takes only 100 tickets though actual things are more.. Is it possible to fetch all available records from JIRA 

### ISSUE THREAD

**Issue by shridhar-tl** [shridhar-tl/jira-assistant#15]
This issue was reported by one of the user and I was not able to reproduce this issue. If anyone else face such issue then +1 this issue.

If you face this issue then if possible provide additional information about it which can include messages / errors from console which could be useful for identifying the issue.

**Comment by Grabes** [comment]
We are experiencing this issue. When making a post request to "/api/2/search" the response contains the field "maxResults: 100", but we have more then 100 issues in the search. 

Here is the request:

```{
"jql":"worklogAuthor in ('ehren.graber') and worklogDate >= '2018-04-21' and worklogDate < '2018-04-29'",
"fields":[
"summary",
"worklog",
"issuetype",
"parent"
],
"maxResults":2000
}```

Here is the response:
```
{
  expand: "schema,names",
  issues: [{expand: "operations,versionedRepresentations,editmeta,changelog,renderedFields", id: "15829",…},…],
  maxResults: 100,
  startAt: 0,
  total: 195
} 
```
Notice that the maxResults is set to 100 and the total exceeds that amount. It appears that the search API is limiting the total issues that are returned.

**Comment by Grabes** [comment]
Changing maxResults parameter for Jira Cloud REST API - https://confluence.atlassian.com/jirakb/changing-maxresults-parameter-for-jira-cloud-rest-api-779160706.html

Recommended solution is to use pagination to return all results. We typically have 400-500 issues per sprint so this is preventing us from using the report.

**Comment by shridhar-tl** [comment]
Thank you @Grabes for providing the request & response details which would be very much helpful in fixing this issue. I will see to that this issue is resolved within this weekend.

**Comment by shridhar-tl** [comment]
This issue is fixed and is available from version 0.46 onward. In case if someone find any issue with this fix then please add a comment in this ticket along with the details of the issue you faced.

**Comment by shridhar-tl** [comment]
Closing this ticket as no issues were reported against this ticket.
